tests/SkiaSharp.Desktop.Tests/SkiaSharp.Desktop.Tests.csproj

```diff
@@ -81,6 +84,9 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="SharpVk">
+      <Version>0.4.2</Version>
+    </PackageReference>
```

Comment:
You added a new tests project, so should this file have been reverted?

Reply:
But, looking again, I see the build succeeded and you have also signed the other project. Do you still need separate projects?

Reply:
Not entirely sure about how signing works, but I can't seem to make it work with the other project.